CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ elseif (${CMAKE_CXX_COMPILER_ID} STREQUAL "MSVC")
 endif()
 
 # Look up required packages and add the include directory to our include path
-find_package(Boost 1.55.0 COMPONENTS filesystem system)
```

Comment:
AFAIR it is minimum required version.

Reply:
Thank you for your contribution by the way :D

Reply:
I will check it again :)

Reply:
You are right. It is minimum required version.